websites/V/VLR.gg/presence.ts

```diff
@@ -0,0 +1,90 @@
+const presence = new Presence({
+    clientId: "939236497985536070",
+  }),
+  browsingTimestamp = Math.floor(Date.now() / 1000);
+let title: HTMLElement;
+
+let rankinginfo: HTMLElement;
+
+let teamicon: HTMLImageElement;
+let eventicon: HTMLImageElement;
+
+let playername: HTMLElement;
+
+presence.on("UpdateData", async () => {
+  title = document.querySelector("head > title");
+  rankinginfo = document.querySelector(".wf-tmp-h .normal");
+  teamicon = document.querySelector(".wf-avatar.team-header-logo img");
+  eventicon = document.querySelector(".wf-avatar.event-header-thumb img");
+  playername = document.querySelector(".player-real-name.ge-text-light");
```

Comment:
Values of these never change, use constants instead

Reply:
Using consts I can't get an ".src" in line 67 and 76, should I change all of them to consts anyway?